modules/library/src/main/scala/zio/elasticsearch/query/Queries.scala

```diff
@@ -200,6 +200,71 @@ private[elasticsearch] final case class ConstantScore[S](query: ElasticQuery[S],
     )
 }
 
+sealed trait DisjunctionMaxQuery[S] extends ElasticQuery[S] {
+
+  /**
+   * Adds specified queries to the [[zio.elasticsearch.query.DisjunctionMaxQuery]]. Returned documents must match one or
+   * more of these queries.
+   *
+   * @param allQueries
+   *   the queries to be added
+   * @tparam S1
+   *   the type of the sub-queries, for which an implicit [[zio.schema.Schema]] is required
+   * @return
+   *   an instance of the [[zio.elasticsearch.query.DisjunctionMaxQuery]] with queries added.
+   */
+  def queries[S1 <: S: Schema](allQueries: ElasticQuery[S1]*): DisjunctionMaxQuery[S1]
+
+  /**
+   * Adds specified queries to the [[zio.elasticsearch.query.DisjunctionMaxQuery]]. Returned documents must match one or
+   * more of these queries.
+   *
+   * @param allQueries
+   *   the queries to be added
+   * @return
+   *   an instance of the [[zio.elasticsearch.query.DisjunctionMaxQuery]] with queries added.
+   */
+  def queries(allQueries: ElasticQuery[Any]*): DisjunctionMaxQuery[S]
```

Comment:
I think you can remove these two methods, as we already ask user for queries within the creation of query.